src/boot/chid.h

```diff
@@ -36,6 +39,13 @@ typedef struct Device {
                         uint32_t name_offset;          /* nul-terminated string or 0 if not present */
                         uint32_t compatible_offset;    /* nul-terminated string or 0 if not present */
                 } devicetree;
+                struct {
+                        /* Offsets are relative to the beginning of the .hwids PE section.
+                           They are nul-terminated strings when present or 0 if not present */
```

Comment:
Not a matter, but please insert a `*` at the beginning of the line, like other comments.